fli-iam/shanoir-ng
# shanoir-issue#1316 Download dataset naming - special chars on windows

How to test:
- rename a dataset, examination, subject with special chars (/!?^() etc..) and download one and multiple datasets.
- Check that it works on windows. 
Closes #1316
